apps/minifront/src/state/swap.ts

```diff
@@ -187,4 +200,48 @@ export const issueSwapClaim = async (swapCommitment: StateCommitment) => {
   await planBuildBroadcast('swapClaim', req, { skipAuth: true });
 };
 
+/*
+  Price impact is the change in price as a consequence of the trade's size. In SwapExecution, the \
+  first trace in the array is the best execution for the swap. To calculate price impact, take
+  the price of the trade and see the % diff off the best execution trace.
+ */
```

Comment:
@hdevalence note that this is a different method than what you proposed in the original issue. After pairing with @erwanor, this felt more accurate.